dev/com.ibm.ws.microprofile.metrics.common/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops

```diff
@@ -59,6 +59,11 @@ internal.error.CWMMC0006E=CWMMC0006E: An internal error occurred.
 internal.error.CWMMC0006E.explanation=The application server experienced an internal error.
 internal.error.CWMMC0006E.useraction=Gather a set of traces and open a new issue against your application server.
 
+#Information regarding libraryRef attribute taking effect only after server restart
+configurationChange.info.CWMMC0007I=CWMMC0007I: A configuration change was detected for the MicroProfile Metrics feature. If a change was made to the libraryRef attribute, a server restart is required for the change to take effect.
+configurationChange.info.CWMMC0007I.explanation=A dynamic reloading of the files referenced by the libraryRef attribute would not invoke a reprocessing of MicroProfile Metrics classes as CDI beans. This would cause instability with the MicroProfile Metrics runtime.
```

Comment:
maybe something like...?
configurationChange.info.CWMMC0007I.explanation=MicroProfile Metrics requires a server restart in order to change libraries referenced by libraryRef.